src/server/app_render.jsx

```diff
@@ -88,20 +74,11 @@ async function appRender(ctx) {
             userPreferences,
             offchain,
         });
-
-        // Assets name are found in `webpack-stats` file
-        const assets_filename =
-            ROOT +
-            (process.env.NODE_ENV === 'production'
-                ? '/tmp/webpack-stats-prod.json'
-                : '/tmp/webpack-stats-dev.json');
-        const assets = require(assets_filename);
-
         // Don't cache assets name on dev
+        // TODO: Investigate removing this.
         if (process.env.NODE_ENV === 'development') {
```

Comment:
I think originally it might be to get some-kind of hot-reloading of assets for local dev. I think it can be removed as we do not typically change the assets while developing locally and requiring a server-restart is not as onerous as perhaps it once was...

https://nodejs.org/api/modules.html#modules_caching